shared_model/cryptography/ed25519_sha3_impl/internal/ed25519_impl.cpp

```diff
@@ -83,10 +90,13 @@ namespace iroha {
     pubkey_t pub;
     privkey_t priv;
 
-    ed25519_create_keypair(pub.data(), priv.data(), seed.data());
+    ed25519_create_keypair(reinterpret_cast<private_key_t *>(priv.data()),
```

Comment:
Why seed argument is unused? https://github.com/hyperledger/iroha/pull/795/commits/762c0bac979e1e0f608500f905ad673df6af5c56 contained this code before rebase
```C++
keypair_t create_keypair(blob_t<32> seed) {
    keypair_t keypair;

    keypair.privkey = seed;

    ed25519_derive_public_key(
        reinterpret_cast<private_key_t *>(keypair.privkey.data()),
        reinterpret_cast<public_key_t *>(keypair.pubkey.data()));

    return keypair;
  }
```